README.md

```diff
@@ -2,30 +2,48 @@
 
 This repository houses the infrastructure for generating PDC SDKs.
 
-We're using a project called [swagger-codegen](https://github.com/swagger-api/swagger-codegen) to power our SDK, and this repository provides tooling and templates
-to make sdk generation easier in particular target languages. The codegen project supports a [many languages](https://github.com/swagger-api/swagger-codegen/tree/master/modules/swagger-codegen/src/main/resources)
-beyond the ones explicitly defined here.
+We're using a project called [swagger-codegen](https://github.com/swagger-api/swagger-codegen) to power our SDK, and this repository provides tooling and templates to make sdk generation easier in specifically supported languages.
 
 ## Prequisites
 
 - JRE (18+)
 
-## Generating sdks
-To generate the SDKs supported by this project, you will need:
+## Generating SDKs
 
-1. A [swagger](https://swagger.io/) specification located at the project root and named `openapi.json`. The current swagger specification for the PDC can be found in the [service repository](https://github.com/PhilanthropyDataCommons/service/blob/main/src/openapi.json).
+To generate the SDKs supported by this project, you will need a [swagger](https://swagger.io/) specification located at the project root and named `openapi.json`. The current swagger specification for the PDC can be found in the [service repository](https://github.com/PhilanthropyDataCommons/service/blob/main/src/openapi.json).
 
-2. Populated configuration files, for each language, located in the `configuration` directory.
+### Generating official SDKs:
 
-#### Steps:
+1. Clone the repository:
 
-From the project root you can run the following commands:
+```shell
+git clone git@github.com:PhilanthropyDataCommons/sdk.git
+```
+
+2. Change directory:
+
+```shell
+cd sdk
+```
+
+3. Copy your swagger specification
+
+```shell
+cp YOUR_SWAGGER_SPEC_LOCATION ./openapi.json
+```
+
+4. Generate the code bases
```

Comment:
```suggestion
4. Generate the codebases:
```

Reply:
<img width="262" alt="image" src="https://github.com/PhilanthropyDataCommons/sdk/assets/208884/3059db51-3c45-4179-a13e-12bb9e2f60bf">

:-|

Reply:
I decided to ditch "codebase" or "code base" entirely

Reply:
…weird. I'd say that's a dictionary with insufficient domain knowledge!

FWIW Wikipedia implies both are correct — "codebase (or code base)" — but goes on to use "codebase" throughout the article: https://en.wikipedia.org/wiki/Codebase

Reply:
yeah the consensus is all of it is ok (including `code-base`) but I just realized the word is longer than needed anyway, I'm not even sure it's the correct term in this case.